ceph-dev-new-trigger/config/definitions/ceph-dev-new-trigger.yml

```diff
@@ -152,12 +152,30 @@
                     DISTROS=centos8
                     FLAVOR=crimson
                     ARCHS=x86_64
+      # build only centos9, no crimson, no jaeger
+      - conditional-step:
+          condition-kind: regex-match
+          regex: .*centos9-only.*
+          label: '${GIT_BRANCH}'
+          on-evaluation-failure: dont-run
+          steps:
+            - shell:
+                !include-raw:
+                - ../../../scripts/build_utils.sh
+                - ../../build/notify
+            - trigger-builds:
+                - project: 'ceph-dev-new'
+                  predefined-parameters: |
+                    BRANCH=${GIT_BRANCH}
+                    FORCE=True
+                    DISTROS=centos9
+                    ARCHS=x86_64
```

Comment:
Nit: I'd move this down (after the "If no release name is found in branch" clause), so that all "build only XYZ" special cases are adjacent.

Reply:
Thanks, I'll move it.